sys-libs/libcxx/libcxx-15.0.7-r1.ebuild

```diff
@@ -121,7 +121,7 @@ multilib_src_configure() {
 		-DLIBCXX_CXX_ABI_INCLUDE_PATHS=${cxxabi_incs}
```

Comment:
I'm confused. Why are you only doing some of the changes to old versions? I understand that this technically improves things, but presumably the whole ebuild doesn't work anyway, so I'd prefer you avoided unnecessary partial changes.

Reply:
Whoops, sorry, I was too trigger-happy with applying this change with `sed`, my fault for not catching it when looking at the diff.

To be precise, this change on itself would still work (it wouldn't break the build), but I agree that this change should be limited to LLVM >= 18.

Reply:
Done